packages/cms-cli/commands/remove.js

```diff
@@ -26,11 +26,44 @@ const {
 } = require('../lib/usageTracking');
 
 const COMMAND_NAME = 'remove';
+const DESCRIPTION = 'Delete a file or folder from HubSpot';
+
+async function action(args, options) {
```

Comment:
One thing that I think we need to decide is what should the signature for the actions be. As a first pass, I split positional arguments from options, but I am not sure if that is the right move or not.